server/mailet/mock-smtp-server/pom.xml

```diff
@@ -119,4 +119,58 @@
             <version>3.1.7</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <configuration>
+                    <from>
+                        <image>openjdk:8u222-jre</image>
+                    </from>
+                    <to>
+                        <image>linagora/mock-smtp-server</image>
+                        <tags>
+                            <tag>latest</tag>
+                        </tags>
+                    </to>
+                    <container>
+                        <mainClass>org.apache.james.mock.smtp.server.MockSMTPServerMain</mainClass>
+                        <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+                        <ports>
+                            <port>25</port>
+                            <port>8000</port>
+                        </ports>
+                    </container>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>docker packaging</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>buildTar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.coderplus.maven.plugins</groupId>
+                <artifactId>copy-rename-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-file</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>rename</goal>
+                        </goals>
+                        <configuration>
+                            <sourceFile>${basedir}/target/jib-image.tar</sourceFile>
+                            <destinationFile>${basedir}/target/jib-mock-smtp-server.tar</destinationFile>
```

Comment:
did you tried the to.image parameter for jib?

Reply:
That's pretty much the section in the code : 
```
<to>
    <image>linagora/mock-smtp-server</image>
    <tags>
        <tag>latest</tag>
    </tags>
</to>
```
It gives that name to the image in the tar, for when you load it under Docker. But you cannot change the archive name.

Did you read the link I posted on one of your similar comments above? Says there is no such conf right now for the output name of a tar build. So for now I'm afraid I don't have a better solution for this. 

However, it seems there is an opened PR about this: https://github.com/GoogleContainerTools/jib/pull/1921